components/builder-protocol/protocols/originsrv.proto

```diff
@@ -15,6 +15,27 @@ message AccountInvitationListResponse {
   repeated OriginInvitation invitations = 2;
 }
 
+enum PackageChannelTrigger {
+  Unknown = 0;
+  BuilderUI = 1;
+  HabClient = 2;
+}
+
+enum PackageChannelOperation {
+  Promote = 0;
+  Demote = 1;
+}
+
+message PackageChannelAudit {
+  optional uint64 package_id = 1;
+  optional uint64 channel_id = 2;
+  optional PackageChannelOperation operation = 3;
+  optional PackageChannelTrigger trigger = 4;
+  optional uint64 requester_id = 5;
+  optional string requester_name = 6;
+  optional string origin_name = 7;
```

Comment:
Is origin name here also meant to be persisted?

Reply:
No, just for routing purposes.

Reply:
👍

Reply:
Although now that I'm thinking about it some more, it's probably appropriate for this to be `origin_id` instead of `origin_name` and for it to be persisted.  I'll update this.